### PR TITLE
Switched to moveit's own dockerfile and fixed dependency repos

### PR DESCRIPTION
**Pull request by CihatAltiparmak**
In rolling, we should upgrade before building workspace due to the fact that rolling is a version that updated continuously. Otherwise docker sometimes can fail. Additionaly, i have fixed some typos in dependency file.

### CODE REVIEW

**Comment by CihatAltiparmak** [comment]
Do you think this updates should go to MoveIt2's docker image? Because seems my PR about deprecation warning throws out build error. ( https://github.com/moveit/moveit2/pull/2933 )

File: moveit_middleware_benchmark.repos [hunk 1]
```diff
@@ -1,5 +1,5 @@
 repositories:
-  moveit_msgs:
+  moveit2:
```
Comment (sjahr) [inline moveit_middleware_benchmark.repos]: Do we need to build moveit2 from source or can't we use the[ available docker images](https://hub.docker.com/r/moveit/moveit2)?

Comment (CihatAltiparmak) [reply moveit_middleware_benchmark.repos]: Hmmm, can be used. I will work on this one in this weekend and then i will ping you again with magical words. Thank you

Comment (sjahr) [reply moveit_middleware_benchmark.repos]: Great thanks!

Comment (CihatAltiparmak) [reply moveit_middleware_benchmark.repos]: I solved it in this commits https://github.com/CihatAltiparmak/moveit_middleware_benchmark/pull/16/commits/f1abbb3527b877127bd7375502a6e33e055f0881 and https://github.com/CihatAltiparmak/moveit_middleware_benchmark/pull/16/commits/16601be7318a30197eda640feb91177d8c0dc37e.

**Comment by CihatAltiparmak** [comment]
Cool mention from me to you (@sjahr ) I'm ready for next review! When you are available, would you take a look at it ? Don't mind CI failures. I believe most of them will be fixed with this PR.

**Review by sjahr — APPROVED** [review]
Thanks! I assume CI failures are not related to the PR. If that is correct go ahead and merge it

**Comment by CihatAltiparmak** [comment]
> Thanks! I assume CI failures are not related to the PR. If that is correct go ahead and merge it

Lets merge this guy then !

**Pull request merged**